doc/src/modules/sets.rst

```diff
@@ -94,3 +94,24 @@ ComplexRegion
    :members:
 
 .. autofunction:: normalize_theta_set
+
+Iteration over sets
+^^^^^^^^^^^^^^^^^^^
+
+For set unions, `\{a, b\} \cup \{x, y\}` may have no problem to be
+comprehended like `\{a, b, x, y\}` regardless of the distinctiveness of
+the elements, however, for set intersections, assuming that
+`\{a, b\} \cap \{x, y\}` is `\varnothing` or `\{a, b \}` would not
+always be valid.
+
+So, we have designed ``__iter__`` method for composite sets to sample an
+element from a enumeratible set and testing its membership for an
```

Comment:
enumeratible -> enumerable / iterable?

Reply:
Iterating over the elements of a set yields (possibly duplicate) elements of the set provided all elements are known to elements of the set. If any element cannot be determined to be a member of a set then iteration gives SomeError. This happens in the same cases where `x in y` would give an error.